doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -308,6 +308,7 @@ DECT NR+ samples
       This is not a full MAC implementation and not fully compliant with DECT NR+ MAC specification (`ETSI TS 103 636-4`_).
     * The ``startup_cmd`` command.
       This command is used to store shell commands to be run sequentially after bootup.
+    * Band 4 support for nrf9151 with modem fw 1.0.2.
```

Comment:
```suggestion
    * Band 4 support for nRF9151 with modem firmware v1.0.2.
```

Reply:
No changes required for this PR but @peknis, you should discuss in doc team what format you prefer as we have both `modem firmware version(s) 1.0.2` and `modem firmware v1.0.2` used in docs and APIs